### PR TITLE
chore: week 43 taxonomy changes

### PR DESCRIPTION
**Pull request by aleene**
### What
More improvements

### CODE REVIEW

File: taxonomies/ingredients_processing.txt [hunk 1]
```diff
@@ -536,6 +536,9 @@ fi:rouhe
 nl:gries, grits, grut
 sv:färs
 
+en:rolled
+nl:geplette
```
Comment (benbenben2) [inline taxonomies/ingredients_processing.txt:540]: <en:agglomerated
en:rolled
de:gerollt, gerollte, gerollter, gerolltes
es:arrollada,arrollado
pt:enrolado, enrolada, enrolados, enroladas

**Review by benbenben2 — CHANGES_REQUESTED** [review]
Everything fine except "rolled" that was already in the taxonomy

**Comment by aleene** [comment]
Ok , I revert the change

**Review by benbenben2 — APPROVED** [review]
All good

**Comment by aleene** [comment]
@stephanegigandet I  can not find the file where the fail occurs. Could this be added to the test log?

**Pull request merged**